app/frontend/app/helpers/notification-message.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export function notificationMessage(notificationObject) {
+  if(notificationObject === undefined) { return; }
+
+  var messageType = notificationObject.get('notificationType');
+  var messageStore = {
+
+    /* Old notification types! */
+    profile_comment: '<user> wrote a post on your feed.',
+    comment_reply: '<user> wrote a reply to your comment.',
+
```

Comment:
Indeed, I should probably remove that, I'm currently mostly using it for testing since this does not contain any backend changes yet.